test/fixtures/mentorships.yml

```diff
@@ -0,0 +1,35 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Got them commented out here. Added helper in test_helper instead. The ULIDS don't allow us to use fixtures.

<img width="654" alt="Screenshot 2023-10-03 at 23 12 40" src="https://github.com/goodscary/firstrubyfriend/assets/29284404/725e19b5-90ac-4cbc-aa30-8e754c1a7b10">